packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -1173,12 +1180,15 @@ private void closeCaptureSession() {
 
   public void close() {
     Log.i(TAG, "close");
-    closeCaptureSession();
```

Comment:
shouldn't this always happen?  if `captureSession != null` shouldn't that session be closed before we set it to `null` ?